docs/source/ContributionGuide.rst

```diff
@@ -228,6 +229,18 @@ Use it::
    You can not use the merge button on github to do signed merges as github
    does not have your private key.
 
+Licensing
+---------
+
+Except where otherwise indicated in a given source file, all original
+contributions to Avocado are licensed under the GNU General Public
+License version 2 `(GPLv2) <https://www.gnu.org/licenses/gpl-2.0.html>`_.
```

Comment:
Yep, it should be GPLv2+. Actually I never understood why do we use GPLv2+, which does not exists, because it means GPLv3+ (once GPLv3 was released the GPLv2+ became GPLv3+) Anyway as I don't care much about that I'm fine with keep using the GPLv2+ term...